post-processor.go

```diff
@@ -106,6 +106,10 @@ func (p *PostProcessor) Configure(raws ...interface{}) error {
 		p.config.ACL = "public-read"
 	}
 
+	if p.config.Region == "us-east-1" {
```

Comment:
Shouldn't this edge case be applied only when formatting the URL? Other code can depend on the region actually being set.